### PR TITLE
request param (ship, take) for GET order api

### PR DESCRIPTION
**Pull request by shyam1s15**
please see details in this issue

https://github.com/ghostfolio/ghostfolio/issues/2374

### CODE REVIEW

File: apps/api/src/app/order/order.controller.ts [hunk 1]
```diff
@@ -89,7 +89,9 @@ export class OrderController {
     @Headers(HEADER_KEY_IMPERSONATION.toLowerCase()) impersonationId,
     @Query('accounts') filterByAccounts?: string,
     @Query('assetClasses') filterByAssetClasses?: string,
-    @Query('tags') filterByTags?: string
+    @Query('tags') filterByTags?: string,
+    @Query('take') take?: number,
+    @Query('skip') skip?: number
```
Comment (dtslvr) [inline apps/api/src/app/order/order.controller.ts:94]: Can you move this line up to keep the alphabetical order?

File: apps/api/src/app/order/order.controller.ts [hunk 2]
```diff
@@ -106,7 +108,9 @@ export class OrderController {
       userCurrency,
       includeDrafts: true,
       userId: impersonationUserId || this.request.user.id,
-      withExcludedAccounts: true
+      withExcludedAccounts: true,
+      skip: isNaN(skip) ? undefined : skip,
```
Comment (dtslvr) [inline apps/api/src/app/order/order.controller.ts:112]: Can you move this line up to keep the alphabetical order?

File: apps/api/src/app/order/order.controller.ts [hunk 3]
```diff
@@ -106,7 +108,9 @@ export class OrderController {
...
+      take: isNaN(take) ? undefined : take
```
Comment (dtslvr) [inline apps/api/src/app/order/order.controller.ts:113]: Can you move this line up to keep the alphabetical order?

File: apps/api/src/app/order/order.service.ts [hunk 4]
```diff
@@ -231,14 +231,18 @@ export class OrderService {
     types,
     userCurrency,
     userId,
-    withExcludedAccounts = false
+    withExcludedAccounts = false,
+    skip,
```
Comment (dtslvr) [inline apps/api/src/app/order/order.service.ts:235]: Can you move this line up to keep the alphabetical order?

File: apps/api/src/app/order/order.service.ts [hunk 5]
```diff
@@ -231,14 +231,18 @@ export class OrderService {
...
+    take = Number.MAX_SAFE_INTEGER
```
Comment (dtslvr) [inline apps/api/src/app/order/order.service.ts:236]: Can you move this line up to keep the alphabetical order?

File: apps/api/src/app/order/order.service.ts [hunk 6]
```diff
@@ -231,14 +231,18 @@ export class OrderService {
...
   }: {
     filters?: Filter[];
     includeDrafts?: boolean;
     types?: TypeOfOrder[];
     userCurrency: string;
     userId: string;
     withExcludedAccounts?: boolean;
+    skip?: number;
```
Comment (dtslvr) [inline apps/api/src/app/order/order.service.ts:244]: Can you move this line up to keep the alphabetical order?

File: apps/api/src/app/order/order.service.ts [hunk 7]
```diff
@@ -231,14 +231,18 @@ export class OrderService {
...
+    take?: number;
```
Comment (dtslvr) [inline apps/api/src/app/order/order.service.ts:245]: Can you move this line up to keep the alphabetical order?

**Review by dtslvr — CHANGES_REQUESTED** [review]

File: apps/api/src/app/order/order.service.ts (hunk 7 above)
Comment (shyam1s15) [reply apps/api/src/app/order/order.service.ts]: changes has been done, thankyou for reviewing :)

File: apps/api/src/app/order/order.service.ts [hunk 8]
```diff
@@ -233,7 +233,7 @@ export class OrderService {
     types,
     userCurrency,
     userId,
-    withExcludedAccounts = false,
+    withExcludedAccounts = false
```
Comment (dtslvr) [inline apps/api/src/app/order/order.service.ts:236]: 👍🏻

**Review by dtslvr — APPROVED** [review]

**Comment by dtslvr** [comment]
Thank you for the changes @shyam1s15! Looks good now 🙂

**Pull request merged**